### DISCUSSION TITLE
Seekbar / progressbar font?

### DISCUSSION THREAD

**Discussion by ppw0** [DeaDBeeF-Player/deadbeef#3013] [Q&A] [answered]
Is there a way to change the font of the time/position that appears during seeking in the center of the seekbar/progress bar? There's an odd serif font in place right now, and I can't find any way to change it.

**Comment by Oleksiy-Yakovenko** [comment, marked as answer]
This specific aspect is not configurable, but it would be interesting to see what the font is and on what platform.
It should be sort of the same font as the rest of UI by default, taken from GTK theme, or NSFont (depending on platform)

**Reply by ppw0** [reply]
Sure, here's an example screenshot.

![Screenshot from 2023-12-06 07-59-53](https://github.com/DeaDBeeF-Player/deadbeef/assets/43313116/aa3204ac-37b8-499f-9e55-385518c3719b)

**Reply by Oleksiy-Yakovenko** [reply]
It looks more like a bug.. It was intended to use the same font as the rest of UI.

**Reply by Oleksiy-Yakovenko** [reply]
(I created a bug from it)
